onnxruntime/core/providers/openvino/openvino_mo/openvino_emitter.py

```diff
@@ -0,0 +1,399 @@
+"""
+  Copyright(C) 2019 Intel Corporation
+  Licensed under the MIT License
+"""
+
+import sys
+import os
+ov_root = os.environ['INTEL_CVSDK_DIR']
```

Comment:
how does this work if you build openvino from source?

Reply:
As we disabled building from source for now, this is hardcoded. This will change once we enable building from source